applications/CMakeLists.txt

```diff
@@ -45,6 +45,12 @@ add_holohub_application(endoscopy_tool_tracking DEPENDS
                                   tool_tracking_postprocessor
                                   OPTIONAL deltacast_videomaster yuan_qcap vtk_renderer)
 
+add_holohub_application(endoscopy_tool_tracking_distributed DEPENDS
+                        OPERATORS lstm_tensor_rt_inference
+                                  tool_tracking_postprocessor
+                                  OPTIONAL deltacast_videomaster yuan_qcap vtk_renderer
+                        HOLOSCAN_SAMPLE_APP
```

Comment:
I believe we no longer consider this flag in HoloHub (@jjomier correct me if I am wrong)
```suggestion
```